App.js

```diff
@@ -31,6 +31,12 @@ class App extends React.Component {
     }
   }
 
+  makeListItem(textToAdd) {
```

Comment:
Where is this method used? (I'm guessing it was from the list item tooltip stuffs?)